lib/ansible/modules/remote_management/irmc/irmc_facts.py

```diff
@@ -0,0 +1,520 @@
+#!/usr/bin/python
+
+# FUJITSU LIMITED
+# Copyright 2018 FUJITSU LIMITED
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: irmc_facts
+
+short_description: get or set PRIMERGY server and iRMC facts
+
+description:
+    - Ansible module to get or set basic iRMC and PRIMERGY server data via iRMC RedFish interface.
+    - For more iRMC modules see https://github.com/FujitsuPrimergy/fujitsu-ansible-irmc-integration.
+
+requirements:
+    - The module needs to run locally.
+    - iRMC S4 needs FW >= 9.04, iRMC S5 needs FW >= 1.25.
+    - Python >= 2.6
+    - Python module 'future'
+
+version_added: "2.8"
+
+author:
+    - Fujitsu Server PRIMERGY (@FujitsuPrimergy)
+
+options:
+    irmc_url:
+        description: IP address of the iRMC to be requested for data.
+        required:    true
```

Comment:
Please use just a single space. If another option is added in the future this will not align
```suggestion
        required: true
```

Reply:
I'll remove the superfluous blanks.